package.json

```diff
@@ -25,12 +26,15 @@
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
     "react-native-elements": "^1.2.7",
     "react-native-gesture-handler": "~1.6.0",
+    "react-native-ionicons": "^5.0.0-rc",
```

Comment:
It looks like this package is not used since you get icons from `'@expo/vector-icons'`, maybe remove it?

Reply:
I am using it to have the fitness icon. I will try to use only one icon package

Reply:
That would be better. We should probably choose the one with the most icons included and the most users on github, so that we're sure to always find what we're looking for and the project won't stop

Reply:
I just had a look and ionicons has 1k users and 25 stars (= likes)
react-native-vector-icons has 155k users and 13k stars, and it includes far more icons (including ionicons)

So maybe we should keep react-native-vector-icons?